<launch> 
    <include file="$(find tiago_iaslab_simulation)/launch/apriltag.launch" /> 
    <include file="$(find tiago_iaslab_simulation)/launch/navigation.launch" /> 
    <node pkg="tiago_iaslab_simulation" type="apriltag_ids_generator_node" name="apriltag_ids_generator_node"/>
</launch>
